### PR TITLE
Feature/round off sorting dialog 8280

### PR DESCRIPTION
**Pull request by tilt**
https://www.openproject.org/work_packages/8280
https://www.openproject.org/work_packages/9836

### CODE REVIEW

File: karma/tests/dialogs/sorting-modal-test.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    beforeEach(function() {
+      scope.sortElements = [
+        [
+          { id: 'parent', label: 'Parent', other: 'Parent' },
+          { id: 'desc', label: 'Descending' }
+        ]
+      ];
+      buildController();
+
+      selectableColumns = scope.getAvailableColumnsData(undefined, angular.identity);
```
Comment (myabc) [inline karma/tests/dialogs/sorting-modal-test.js:117]: @tilt first time I've seen `angular.identity`!

File: karma/tests/dialogs/sorting-modal-test.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+      });
+
+      $timeout.flush();
+    };
+
+  }));
+
+  describe('initialisation', function() {
+    it('should initialise', function() {
+      buildController();
```
Comment (myabc) [inline karma/tests/dialogs/sorting-modal-test.js]: @tilt since this describe block doesn't contain any assertions, we could probably delete it.

Comment (tilt) [inline karma/tests/dialogs/sorting-modal-test.js]: yes go ahead ;) but let's add [ci skip]

File: karma.conf.js [hunk 3]
```diff
@@ -38,6 +38,7 @@ module.exports = function(config) {
       "app/assets/javascripts/angular/config/work-packages-config.js",
 
       "app/assets/javascripts/angular/controllers/**/*.js",
+      "app/assets/javascripts/angular/dialogs/**/*.js",
```
Comment (myabc) [inline karma.conf.js:41]: @tilt this path doesn't exist: it's `app/assets/javascripts/angular/controllers/dialogs/**/*.js`

**Comment by myabc** [comment]
:+1: working for me.

**Pull request merged**